variables.tf

```diff
@@ -0,0 +1,148 @@
+/**
```

Comment:
The variable descriptions in this file need some work -- they should be easily readable and understandable to new users of the module.